recipe/meta.yaml

```
@@ -1,39 +1,29 @@
 {% set name = "conda-package-handling" %}
-{% set version = "1.9.0" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/conda-package-handling/archive/{{ version }}.tar.gz
-  sha256: 81e0771b8b10ab5c20678a5da714e36d97e321acfc3e5b304ec2e650ce84734b
+  sha256: 7b10976cd8aa09307c41e9f19795347b44e83d22077519dc8f3cd7778f68755c
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - cph = conda_package_handling.cli:main
-  # HACK: case-sensitive mismatch in latest conda-build
-  missing_dso_whitelist:
-    - '*/VCRUNTIME140.dll'   # [win]
-    - MSVCR90.dll     # [win]
+  noarch: python
```

As this is such an important central package: can we temporarily remove noarch + rerender to see it successful building (+testing) on all platforms + all python versions? This proves that the package itself  plus its dependencies work properly there.
```suggestion
  #noarch: python
```

Once CI parses the all CI runs, I would be fine with making it noarch again in this PR.